controllers/C_Prescription.class.php

```diff
@@ -40,6 +40,54 @@ function __construct($template_mod = "general")
         $this->pconfig = $GLOBALS['oer_config']['prescriptions'];
         $this->RxList = new RxList();
 
+        $interaction = "";
+
+        /*
+        *   check to see if RxNorm installed
+        */
+        $rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = 'RXNCONSO' OR table_name = 'rxnconso'");
```

Comment:
you should be able to do this instead:
$rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = ?", array(mitigateSqlTableUpperCase('RXNCONSO'));